pkg/service/configmap_env_builder.go

```diff
@@ -58,6 +58,7 @@ func (n *EnvBuilder) buildRedisEnv() (map[string]string, error) {
 	data["USE_STATSD"] = "false"
 	data["REDIS_TYPE"] = n.RateLimitService.Spec.Backend.Redis.Type
 	data["REDIS_URL"] = n.RateLimitService.Spec.Backend.Redis.URL
+	data["REDIS_AUTH"] = n.RateLimitService.Spec.Backend.Redis.Auth
```

Comment:
not all Redis implementation is using auth, can we check if `n.RateLimitService.Spec.Backend.Redis.Auth` and assign to data map if it's exist?